element_calcium_imaging/scan.py

```diff
@@ -443,41 +444,40 @@ def estimate_nd2_scan_duration(nd2_scan_obj):
                 )
         elif acq_software == "PrairieView":
             from element_interface import prairieviewreader
-            
 
             pvscan_filepaths = get_prairieview_files(key)
             pvscan_info = prairieviewreader.get_pv_metadata(pvscan_filepaths[0])
-
             self.insert1(
                 dict(
                     key,
-                    nfields = pvscan_info["num_fields"],
-                    nchannels = pvscan_info["num_nchannels"],
-                    ndepths = pvscan_info["num_planes"],
-                    nframes = pvscan_info["num_frames"],
-                    rois = pvscan_info["num_rois"],
-                    x = pvscan_info["x_pos"],
-                    y = pvscan_info["y_pos"],
-                    z = pvscan_info["z_pos"],
-                    fps = pvscan_info["frame_rate"],
-                    usecs_per_line = pvscan_info["usecs_per_line"],
-                    scan_datetime = pvscan_info["scan_datetime"],
-                    scan_duration = pvscan_info["scan_duration"],
+                    nfields=pvscan_info["num_fields"],
```

Comment:
Isn't `nfields` already in `ScanInfo`?

Reply:
We're populating the `nfields` in `ScanInfo` here

Reply:
oops. Got it.